lib/angular-evaporate.js

```diff
@@ -301,7 +302,9 @@
                * @returns - same as Evaporate.add()
                */
               AngularEvaporate.prototype.$start = function (upload) {
-
+                var args = Array.prototype.slice.call(arguments);
+                args[1] = upload.overrideConfig;
```

Comment:
My bad, I must've screwed up my commit somehow. `args` is meant to be passed in to this.add, instead of `arguments`. Ideally I'd like to just set `arguments[1]` instead of doing a shallow copy, but because `arguments` is a special type and not actually an array it seems that this doesn't work.